examples/client/index.html

```diff
@@ -88,8 +88,34 @@
 						return "Error: 'value' in 'where' clause is missing.";
 					}
 
-					// todo: find out how to decode bigint field types. i.e. what is a 34n?
-					// assert `where` type
+					const allowedRanks = fieldTypes[whereNode.key];
+
+					let valueRank;
+					let valueType;
+
+					if (whereNode.value === null) {
+						valueRank = 1;
+						valueType = "null";
+					}
```

Comment:
This check can be simplified a bit by directly checking the bits:

```suggestion
const FIELD_TYPE_NULL = (1 << 0);

...

					if (whereNode.value === null) {
					  if ((field.fieldType & FIELD_TYPE_NULL) > 0) {
					    return <error>;
					  }
					}
```